front/src/modules/companies/components/AddPersonToCompany.tsx

```diff
@@ -88,12 +133,30 @@ export function AddPersonToCompany({
 
         {isDropdownOpen && (
           <div ref={refs.setFloating} style={floatingStyles}>
-            <PeoplePicker
-              personId={''}
-              onSubmit={handlePersonSelected(companyId)}
-              onCancel={handleClosePicker}
-              excludePersonIds={peopleIds}
-            />
+            {isCreationDropdownOpen ? (
+              <StyledInputContainer>
+                <TextInput
+                  onKeyDown={handleInputKeyDown}
+                  value={username.firstName}
+                  onChange={handleUsernameChange('firstName')}
+                  placeholder="First Name"
+                />
+                <TextInput
+                  onKeyDown={handleInputKeyDown}
+                  value={username.lastName}
+                  onChange={handleUsernameChange('lastName')}
+                  placeholder="Last Name"
+                />
```

Comment:
In order to align the appearance of the `DoubleTextInput` with the design specifications, significant styling adjustments were necessary. As a solution, we opted against overriding the existing styles and instead utilized the `TextInput` components directly.

Reply:
The video specifications mandated that the button remain consistently positioned at the bottom of the people list. Nonetheless, the current functionality of the `SingleEntitySelect` differs, displaying the button solely when filters are populated. We chose not to modify this behavior.